cfme/tests/configure/test_access_control.py

```diff
@@ -136,20 +136,26 @@ def test_user_change_groups(appliance, request, group_collection):
     """Assign a user to multiple groups and confirm that the user can successfully change groups
     """
     group_names = [
-        'EvmGroup-user', 'EvmGroup-user_self_service', 'EvmGroup-desktop']
+        'EvmGroup-super_administrator', 'EvmGroup-administrator', 'EvmGroup-approver',
+        'EvmGroup-auditor', 'EvmGroup-desktop', 'EvmGroup-operator',
+        'EvmGroup-security', 'EvmGroup-user', 'EvmGroup-vm_user', ]
 
-    group_list = [group_collection.instantiate(description=names) for names in group_names]
+    group_list = [group_collection.instantiate(description=name) for name in group_names]
```

Comment:
I like this approach.   I would have simply tried to change the group back as a separate line of code after the loop.   This is much more elegant.